spec/support/shared_examples/shared_not_found.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.shared_examples "trying to access another user's resource" do
```

Comment:
Since this shared example is only used by a single file (`spec/requests/api/bets_spec.rb`) it is best practice to keep it in the file that uses it.

If you wish to make it a more general example, you can always do that when necessary.